.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,21 @@
+name: Unit Tests for Charts
+
+on:
```

Comment:
You can run this workflow on the PR too instead of duplicating it in `tests-pr.yaml`.
Can rename `tests-pr.yml` file to `smoke-lint-tests.yml` or something similar to make it clear that it contains smoke and lint tests.

Reply:
thank you. I updated as you suggested with 4e7333ce68c57f575e66fe36a666a7cd63d38b7a.